File: 01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/Volleyball.java
import java.util.Scanner;

public class Volleyball {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String year = scanner.nextLine();
        int holidays = Integer.parseInt(scanner.nextLine());
        int homedays = Integer.parseInt(scanner.nextLine());

        double inSofia = (48 - homedays) * 3/4.0;
        double inHome = homedays;
        inSofia += (holidays * 2/3.0);
        double totalGames = inSofia + inHome;
        if (year.equals("leap")) {
            totalGames += (totalGames * 0.15);
        }

        System.out.printf("%.0f", Math.floor(totalGames));
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/Volleyball.js
function volleyball([arg1, arg2, arg3]) {
    let yearType = arg1;
    let holidays = Number(arg2);
    let weekendsHome = Number(arg3);

    let totalGames = 0;
    let gamesSofia = ((2 / 3) * holidays) + (3 / 4) * (48 - weekendsHome);
    let gamesHome = weekendsHome;

    totalGames = gamesHome + gamesSofia;

    if (yearType === 'leap') {
        totalGames *= 1.15;
    }

    console.log(Math.floor(totalGames));
}

volleyball(['leap', '5', '2']);
volleyball(["normal", "3","2"]);